.github/workflows/test.yml

```diff
@@ -1,6 +1,11 @@
 name: Test
 on:
-- pull_request
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
This makes it so the workflow is triggered on pull requests to `main` and on pushes to `main`.  If you want to run on pull requests to any branch, it could be updated to look like:
```yaml
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - *
```

(Or the `branches` property could be omitted altogether from the `pull_request` object.)

Reply:
I see, that sounds good 👍

Reply:
Actually I do think we want to run the workflow on any pull requests, we often split the work into multiple branches, only the top-most being derived from `main`.

Reply:
Latest commit is updated to run on pull requests to all branches.